react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
The original comment was: "This placeholder is not the only thing passed from C++, so it should be PAGE_TITLE_FROM_C_PLUS_PLUS." (currently just FROM_C_PLUS_PLUS). I saw you did DATA_JSON to DATA_FROM_C_PLUS_PLUS change which is OK, but I can't see the same change for PAGE_TITLE.